plugin/src/main/java/io/jenkins/plugins/analysis/warnings/QtTranslation.java

```diff
@@ -0,0 +1,66 @@
+package io.jenkins.plugins.analysis.warnings;
+
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.parser.QtTranslationParser;
+
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import org.jenkinsci.Symbol;
+
+import hudson.Extension;
+
+import io.jenkins.plugins.analysis.core.model.ReportScanningTool;
+
+public class QtTranslation extends ReportScanningTool {
+    private static final long serialVersionUID = 1L;
+    private static final String ID = "Qt-Translation";
```

Comment:
Since this will be a URL it would be better to use `qt` or `qt-translation` (all lowercase).

Reply:
Used `qt-translation` and not `qt` in case there will be more Qt related parser (don't know if there is e.g. an QML validator or stg. like this).